hack/jenkins/e2e.sh

```diff
@@ -430,6 +469,8 @@ case ${JOB_NAME} in
           ${GCE_DEFAULT_SKIP_TESTS[@]:+${GCE_DEFAULT_SKIP_TESTS[@]}} \
           ${GCE_FLAKY_TESTS[@]:+${GCE_FLAKY_TESTS[@]}} \
           )"}
+    : ${JENKINS_GCS_LOGS_PATH:="gs://kubernetes-jenkins/logs"}
```

Comment:
wasn't sure if we also wanted all GKE builds uploaded, but I guess it's harmless enough.